### PR TITLE
Update benchmarking process in RELEASE.md

### PR DESCRIPTION
**Pull request by Ngoguey42**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -25,32 +25,75 @@ described in the instructions below.
...
+before running the benchmarks.
+
+The benchmark should be run at least twice (to give insights about the noise), 
+the two resulting `stat_trace.repr` files should be copied to 
+`/bench/releases/` using the same naming convention as above. Also set the file
+permissions to read-only using `chmod 444 stat_trace.repr`.
+
+A run of the benchmark is expected to last \~35min.
+
+See [this script](https://github.com/tarides/irmin-tezos/blob/master/bench.sh) for an example of a setup and a bench run.
```
Comment (CraigFe) [inline RELEASE.md:70]: This repository is private currently.

File: RELEASE.md [hunk 2]
```diff
@@ -25,32 +25,75 @@ described in the instructions below.
...
+            ├── stat_summary.json
+            └── stat_trace.repr
+```
 
-From the benchmarking server, setup the right versions of the right libraries
-and then run Irmin's trace replay:
-```sh
-; export ROOT='/bench/releases/irminX.Y.Z-indexX.Y.Z-reprX.Y.Z'
-; dune exec -- bench/irmin-pack/tree.exe --mode trace --ncommits-trace 100000 /bench/replay-traces/tezos-replayable-v0.repr --verbosity info --path-conversion v0+v1 --artefacts $ROOT --keep-stat-trace
+To test a new release, setup an `c3-small-x86-01` aws instance, fetch a large
```
Comment (CraigFe) [inline RELEASE.md:59]:
```suggestion
To test a new release, setup an `c3-small-x86-01` Equinix instance, fetch a large
```
?

File: RELEASE.md (hunk 1 above)
Comment (Ngoguey42) [reply RELEASE.md:70]: Yes. I think this is okey since only people with access to that repo play a role in the release processes

Comment (CraigFe) [reply RELEASE.md:70]: OK :+1:

**Comment by CraigFe** [comment]
Thanks!

**Pull request merged**